sysinit/coqinit.ml

```diff
@@ -182,11 +182,22 @@ let init_runtime opts =
   | Coqargs.Run ->
       injection_commands opts
 
-let require_file ~intern ~prefix ~lib ~export =
+let warn_require_not_found =
+  CWarnings.create ~name:"command-line-require-not-found"
+    ~category:CWarnings.CoreCategories.filesystem
+    Pp.(fun (prefix, lib) ->
+        strbrk "Cannot require library " ++ Libnames.pr_qualid lib ++
```

Comment:
"Cannot require" is not very clear. "Did not find" or something like that would be clearer IMHO. And since this warning is only raised when using `-compat` options, what about "Did not find compatibility module..."?